stdlib/public/Concurrency/TaskGroup.swift

```diff
@@ -63,61 +69,50 @@ extension Task {
   public static func withGroup<TaskResult, BodyResult>(
     resultType: TaskResult.Type,
     returning returnType: BodyResult.Type = BodyResult.self,
-    body: @concurrent @escaping (inout Task.Group<TaskResult>) async throws -> BodyResult
+//    startingChildTasksOn executor: ExecutorRef? = nil, // TODO: actually respect it
```

Comment:
this is coming as soon as we land custom executors and those proposals and discuss this on evolution but seems likely